target.py

```diff
@@ -0,0 +1,24 @@
+import pygame
+import random
+
+
+class Target(pygame.sprite.Sprite):
+
+    def __init__(self, width, height):
+        pygame.sprite.Sprite.__init__(self)
+        self.image = pygame.Surface((width, height))
+        self.image = pygame.image.load('target.png')
```

Comment:
This is regarding project structure.  I think we should put all images and sprites into a resource folder.  Later on if we add audio, audio files can go into the resource folder too.